### PR TITLE
remove beta flag from GUI Editor

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13726/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13726/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13726/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13726/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13726/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13726/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13726/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13726/merge#BCU1XR#0

**Review by sebavan — APPROVED** [review]

File: packages/tools/guiEditor/src/components/commandBarComponent.tsx [hunk 1]
```diff
@@ -338,9 +337,7 @@ export class CommandBarComponent extends React.Component<ICommandBarComponentPro
                         )}
                     </div>
                 </div>
-                <div className="commands-right">
-                    <img src={betaFlag} className="beta-flag" draggable={false} />
-                </div>
+                <div className="commands-right"></div>
```
Comment (sebavan) [inline packages/tools/guiEditor/src/components/commandBarComponent.tsx:340]: Should we still keep it ?

Comment (carolhmj) [reply packages/tools/guiEditor/src/components/commandBarComponent.tsx:340]: I thought to keep it if we needed to put anything later and also to not risk messing with the existing CSS :P

**Pull request merged**